easypost/shipment.py

```diff
@@ -0,0 +1,84 @@
+from easypost.error import Error
+from easypost.resource import AllResource, CreateResource, Requestor
+
+
+class Shipment(AllResource, CreateResource):
+    @classmethod
+    def track_with_code(cls, api_key=None, **params):
```

Comment:
This `track_with_code` function is super dead. Please remove it here and add a note to the CHANGELOG.